lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -1011,4 +1015,25 @@ std::string SPIRVToLLVMDbgTran::findModuleProducer() {
   return "spirv";
 }
 
+Optional<DIFile::ChecksumInfo<StringRef>>
+SPIRVToLLVMDbgTran::ParseChecksum(StringRef Text) {
+  // Example of checksum string:
+  // "//__CSK_MD5:7bb56387968a9caa6e9e35fff94eaf7b"
+  Optional<DIFile::ChecksumInfo<StringRef>> CS;
+  auto KindPos = Text.find(SPIRVDebug::ChecksumKindPrefx);
+  if (KindPos != StringRef::npos) {
```

Comment:
As "//__CSK" is a prefix, I suggest to use [startswith](https://llvm.org/doxygen/classllvm_1_1StringRef.html#aeb540aeadd339af860f5c217a85363b7) here.

Reply:
The variable "Text" can start with a different prefix and contain information about the checksum, for example, in the middle, so we need to find ChecksumKindPrefx.

Reply:
Added a comment to the code to clarify this point.